packages/core/src/index.ts

```diff
@@ -140,6 +140,7 @@ export { Stamp, type StampProps } from './Stamp/Stamp'
 export { Step, type StepProps } from './Step/Step'
 export { Stepper, type StepperProps } from './Stepper/Stepper'
 export { Swatch, type SwatchProps } from './Swatch/Swatch'
+export { Tab, TabDataProps, type TabProps } from './Tab/Tab'
```

Comment:
Any reason to exclude type from `TabDataProps`?
```suggestion
export { Tab, type TabDataProps, type TabProps } from './Tab/Tab'
```